ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -77,6 +77,12 @@ releases:
       product: ocp
       stream: nightly
       version: "4.16"
+  source:
```

Comment:
openshift-e2e-test-qe does not reference openshift-extended-upgrade-pre, so we do not need the setting here.

Reply:
it has cucushift-sdn-live-migration-ovn, cucushift-sdn-migration-ovn and openshift-upgrade-qe-test-heterogeneous which have pre step, so keep it.